### DISCUSSION TITLE
Really promising library

### DISCUSSION THREAD

**Discussion by husayt** [jd-solanki/anu#98] [General]
Hi, Just wanted to post this as an encouragement for this project. UnoCss and VueUse are yet to be widely discovered gems and what is needed next is to have a UI library built on top. Then we have a really nuclear (pun on atomic css) stack to build top efficient projects. 

Obviously there is still a lot of work ahead, but I am sure this a right direction and once mature will get a lot more attention and has the potential to be the library to go for Vue based projects

Thanks for this initiave and keep the effort

**Comment by jd-solanki** [comment]
Thanks for the encouragement, This means a lot to Anu ☺️
